### DISCUSSION TITLE
ItemEntity right-click

### DISCUSSION THREAD

**Discussion by playgamesgo** [Minestom/Minestom#2452] [Help] [answered]
How to determine that player right-clicks an ItemEntity? PlayerEntityInteractEvent works only with regular entities.

**Comment by cosrnic** [comment, marked as answer]
you might have to spawn an interaction entity and use that to detect right clicks, you could probably add it as a passenger to the item entity and change its size to be similar. You may also need to set the bounding box of the entity although im not sure

**Reply by playgamesgo** [reply]
Ok, I understand, thank you
